### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TymekGluch**
adding html structure and css simple styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                </p>
+
+                <button id="start-button" class="main-header__start-button" aria-label="start game">
+                    start
+                </button>
+            </header>
+
+            <section class="game-section">
+                <div class="game-board"> 
+                    <button id="field-1.1" class="game-board__field game-board__field--cross" disabled aria-label="field - 1.1"></button>  <!-- field - row.column -->
```
Comment (MarekGluch) [inline index.html:29]: at the moment I have no idea if these id's will be needed

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                    <button id="field-2.3" class="game-board__field" aria-label="field - 2.3 "></button>
+                    <button id="field-3.1" class="game-board__field" aria-label="field - 3.1"></button>
+                    <button id="field-3.2" class="game-board__field" aria-label="field - 3.2"></button>
+                    <button id="field-3.3" class="game-board__field" aria-label="field - 3.3"></button>    
+                </div>
+                
+            <button id="refresh-button" class="game-section__refresh-button" aria-label="refresh game"></button>
+            </section>
+            <footer class="footers">
+                <p class="footers__text">
```
Comment (MarekGluch) [inline index.html:43]: Are you sure two paragraphs were needed here?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+.main-header__start-button {
+    border-radius: 50%;
+    width: 48px;
+    height: 48px;
+    border: none;
+    background-color: var(--accent-color);
+    color: var(--text-icons-color);
+    font-family: 'Montserrat', sans-serif;
+    font-size: medium;
+    font-weight: 900;
```
Comment (MarekGluch) [inline style.css:62]: you only import 100 and 300 weight and here you use 900

File: style.css [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    flex-direction: column;
+}
+.footers__text {
+    margin-left: 5px;
+    font-family: 'Montserrat', sans-serif;
+    color: var(--primary-text-color);
+    font-size: 16px;
+}
+
+
```
Comment (MarekGluch) [inline style.css:168]: this break is not needed

File: style.css [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+    no-repeat center;
+}
+.game-section__refresh-button:hover {
+    transform: rotate(720deg);
+    transition-duration: 350ms;
```
Comment (MarekGluch) [inline style.css:93]: Always indicate which CSS property is to be animated (it is good practice to only animate transform and opacity, if possible)

**Review by MarekGluch — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (TymekGluch) [reply index.html:29]: i did it

File: index.html (hunk 2 above)
Comment (TymekGluch) [reply index.html:43]: i changed it

File: style.css [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+    font-family: 'Montserrat', sans-serif;
+}
+.main-header__start-button {
+    border-radius: 50%;
+    width: 48px;
+    height: 48px;
+    border: none;
+    background-color: var(--accent-color);
+    color: var(--text-icons-color);
+    font-family: 'Montserrat', sans-serif;
```
[the remark this answers is not in the document]
Comment (TymekGluch) [reply style.css:60]: i chenged it

File: style.css (hunk 3 above)
Comment (TymekGluch) [reply style.css:62]: i changed it

File: style.css (hunk 5 above)
Comment (TymekGluch) [reply style.css:93]: i changed it

File: style.css [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+    border-radius: 50%;
+}
+.game-board__field--cross::before,
+.game-board__field--cross::after {
+    width: 5px;
+    background-color: var(--primary-text-color);
+}
+.game-board__field--cross::before {
+    transform: translate(-50%, -50%) rotate(45deg);
+
```
[the remark this answers is not in the document]
Comment (TymekGluch) [reply style.css:139]: i changed it

File: style.css [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+.game-board__field:hover {
+   opacity: 0.5;
+}
+.game-board__field:disabled {
+   opacity: 1;
+   cursor: not-allowed;
+}
+
+.footers {
+    height: 7VH;
```
[the remark this answers is not in the document]
Comment (TymekGluch) [reply style.css:153]: i changed all capital letters  to lower case

File: style.css (hunk 4 above)
Comment (TymekGluch) [reply style.css:168]: i changed it

**Review by GabrielGladykowski — APPROVED** [review]

**Review by MarekGluch — APPROVED** [review]

**Pull request merged**